app/manifest.json

```diff
@@ -7,7 +7,15 @@
   "background": {
     "scripts": ["background.js"]
   },
-  "browser_action": {
+  "content_scripts": [
+    {
+      "matches": ["http://*/*", "https://*/*"],
+      "js": ["content.js"],
+      "run_at" : "document_idle"
+    }
+  ],
+  "web_accessible_resources": ["lib/sinon.js", "lib/jquery.js"],
```

Comment:
Can you tell the reason for doing this?

Reply:
I am injecting jquery and sinon to DOM. https://github.com/code-mancers/interceptor/blob/c960288208613e7beffa1b1ada63f371a87401eb/app/content.ts#L25-L33 . Content-scripts are loaded in a context different from DOM's context, So packed those scripts in extension itself rather than using a CDN. And again, I could avoid jquery for making Ajax requests

Reply:
I understand you want these scripts to run, but won't webpack automatically bundle these together?

Reply:
Ah, okay. this usage is different.

Reply:
However, there's a potential problem. What if the target URL already includes jquery? possibly of a different version?

Reply:
Yeah, that's something to consider. Maybe lets avoid jquery completely as it becomes too heavy for our use case and use something like axios and check for conflicts before injecting?